From 1ee2ac79b47eae725bd8d1d54adfaa06086ebae8 Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
Date: Mon, 21 Oct 2013 20:29:12 +0200
Subject: [PATCH] background: Fix compilation with libsocialweb

When compiling with libsocialweb:
./.libs/libbackground-chooser.a(cc-background-chooser-dialog.o): In
  function `cc_background_chooser_dialog_init':
/home/sjoerd/Debian/build-area/gnome-control-center-3.10.1/panels/background/cc-background-chooser-dialog.c:282:
  undefined reference to `bg_flickr_source_new'

Ensuring the flicker-source source files are part of
libbackground-chooser.la fixes this
---
 panels/background/Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/panels/background/Makefile.am b/panels/background/Makefile.am
index 2766207..58e39aa 100644
--- a/panels/background/Makefile.am
+++ b/panels/background/Makefile.am
@@ -49,9 +49,9 @@ libbackground_la_SOURCES =		\
 libbackground_la_LIBADD = $(PANEL_LIBS) $(BACKGROUND_PANEL_LIBS) libbackground-chooser.la
 
 if WITH_LIBSOCIALWEB
-libbackground_la_SOURCES += bg-flickr-source.c bg-flickr-source.h
+libbackground_chooser_la_SOURCES += bg-flickr-source.c bg-flickr-source.h
 INCLUDES += $(SOCIALWEB_CFLAGS)
-libbackground_la_LIBADD += $(SOCIALWEB_LIBS)
+libbackground_chooser_la_LIBADD += $(SOCIALWEB_LIBS)
 endif
 
 noinst_PROGRAMS = test-chooser-dialog
-- 
1.8.4.rc3

